pyaedt/edb_core/edb_data/simulation_configuration.py

```diff
@@ -114,9 +118,27 @@ def dielectric_extent(self, value):  # pragma: no cover
         if isinstance(value, (int, float)):
             self._dielectric_extent = value
 
+    @property
+    def use_dielectric_extent_multiple(self):
+        """Retrieve the dielectric extent multiple value.
+
+        Returns
+        -------
+            bool
+            Multiple value, ``True```will use multiple e.g. extent factor, ``False`` will use absolute dimension.
+            Default value is ``True``.
```

Comment:
```suggestion
        """Whether the multiple value of the dielectric extent is used.

        Returns
        -------
        bool
           ``True`` when the multiple value (extent factor) is used, ``False`` when the absolute dimension is used.
```
This is a property. It should be a noun description. Doesn't it just return whatever value has been set elsewhere? I don't think a default value needs to be specified here.